src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -1055,3 +1063,20 @@ const AlterationMenuHeader: React.FunctionComponent<{
         );
     }
 );
+
+export function getSampleMolecularIdentifiers(
+    selectedSamples: Sample[],
+    profiles: MolecularProfile[]
```

Comment:
if there are multiple profiles per study, this won't work. determine if that's really the case here.  (whether there can be multiple profiles)